lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -98,6 +101,8 @@ RESET_IDENTITY:
   response:
     type: process
     lambda: reset-identity
+    lambdaInput:
+      isUserInitiated: false
```

Comment:
Just checking does this have to be provided? (or can it just default to false?)

Reply:
I think it does have to be provided